stories/data-table.stories.js

```diff
@@ -32,13 +32,26 @@ import { __, _x } from '@wordpress/i18n';
  */
 import Layout from '../assets/js/components/layout/layout';
 import AnalyticsDashboardWidgetTopPagesTable from '../assets/js/modules/analytics/components/dashboard/AnalyticsDashboardWidgetTopPagesTable';
+import { dashboardPopularPagesArgs, dashboardPopularPagesData } from '../assets/js/modules/analytics/datastore/__fixtures__';
 import { googlesitekit as analyticsDashboardData } from '../.storybook/data/wp-admin-admin.php-page=googlesitekit-module-analytics-googlesitekit';
+import { STORE_NAME as MODULES_ANALYTICS } from '../assets/js/modules/analytics/datastore/constants';
 import { WithTestRegistry } from '../tests/js/utils';
 
 storiesOf( 'Global', module )
 	.add( 'Data Table', () => {
 		global._googlesitekitLegacyData = analyticsDashboardData;
 
+		const setupRegistry = ( { dispatch } ) => {
+			dispatch( MODULES_ANALYTICS ).receiveGetSettings( {
+				accountID: '123456789',
+				propertyID: '123456789',
+				internalWebPropertyID: '123456789',
+				profileID: '123456789',
```

Comment:
Are these values necessary at all? If not the Analytics datastore fixtures has a `settings.default` which would be good to provide valid (but empty) defaults. Otherwise, the `propertyID` is not a valid format and should be `UA-#######-#`

Reply:
Ah, good shout. I actually copied these from another storybook test. I'll take a quick look to see if these—and the other test—can use `settings.default`. They probably can.

Reply:
Ah, they can't be empty, just tried. But I'll use valid values, cheers 👍🏻